omc3/tune_analysis/timber_extract.py

```diff
@@ -2,29 +2,35 @@
 Timber Extraction
 -----------------
 
-Tools to extract data from ``Timber``.
+Tools to extract data from ``Timber``. It is a bit heavy on the LHC side at the moment.
```

Comment:
Did you put an issue or something somewhere to remember to remove this comment when things will go back to a normal state?

Reply:
The comment is not from me and has only been moved a few lines up, so I'm not sure. As far as I know this code is initially from @JoschD

Reply:
I only have LHC experience with Timber and I am not sure if we will use it with any other machine soon. It's only for BBQ-cleaning anyway. So LHC is the "normal state"

Reply:
It is meant as a comment: If you have any other machine and you think you can extract data from timber for it... you need to implement that first.